sehee/controllers/likesController.js

```diff
@@ -0,0 +1,19 @@
+const likesService = require("../services/likesService");
+
+const createLike = async (req, res) => {
+  try {
+    const { postId, userId } = req.params;
+    if (!postId || !userId) {
+      return res.status(400).json({ message: "INVALID_DATA_INPUT" });
+    }
+    await likesService.createLike(postId, userId);
+    return res.status(201).json({ message: "likeCreated" });
+  } catch (err) {
+    console.log(err);
```

Comment:
`console.log(err);` 로 에러가 발생했을 때 로그를 확인하는 좋은 코드입니다!

다만 다른 특정 로그와 헷갈릴 수 있기 때문에 아래와 같이 작성할 수 있다는 것도 참고해주세요!

 ```suggestion
    console.error(err);
```